packages/cx-editors/src/components/RichText/tokens/cxRichText.ts

```diff
@@ -109,26 +111,43 @@ const Default = asCxTokenSpec()({
   },
 });
 
-const Copyright = asCxTokenSpec()({
+const Basic = asCxTokenSpec()({
   ...Default,
+  Core: pick(Default.Core, 'paragraph', 'Bold', 'Underline', 'Link', 'SuperScript'),
+  Components: pick(Default.Components, 'Bold', 'Underline', 'SuperScript'),
```

Comment:
Can you change Default.Theme
Then in default change the the Componets -> Theme -- that was my miss.

Reply:
Changed from Components to Theme